sdcm/nemesis.py

```diff
@@ -1171,8 +1172,10 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=MAX_TIME_WAIT
         self.monitoring_set.reconfigure_scylla_monitoring()
         self.set_current_running_nemesis(node=new_node)  # prevent to run nemesis on new node when running in parallel
         new_node.replacement_node_ip = old_node_ip
+        hard_timeout = soft_timeout + HOUR_IN_SEC * 3  # Setting a hard-timeout with an extra 3 hours
```

Comment:
Would be a bit more useful if the caller of the function can set both soft and hard

Reply:
Also can we limit the change to the replace operation, and not change the timing for all of the nemesis that add a node ?

Reply:
added `hard_limit`.

As for limiting the change - there is no option and there's no need doing it. and, in addition, the operation is not what matters here but the data-set size.
So if a new nemesis wish a different value - it could easily be set by the parameters.

Reply:
That not exactly accurate, since currently we have RBO on specific operation and not on all of them.

Anyhow keeping a very large timeouts, basically mean that if we are gonna have an issue that is starts slowing, we'll never gonna notice it in small data set, and we'll also figure it out really really late, like now when we wanted to release.

Reply:
ok, fixed - it is now increased only for replace-node.